releasenotes/notes/fix-botocore-stepfunction-patching-9993630697956278.yaml

```diff
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    botocore: This fix resolves a regression where trace context was not being injected into the input of Stepfunction
```

Comment:
Is the result of this distributed tracing across stepfunctions? If so, can we add that?

Reply:
The result of this is that, when a Python service invokes a Step Function, we can properly parent the step function's trace to the Python service's trace. Do you want to (re-)add that as a feature, or should I just mention it as a fix?